ofl/lxgwwenkaitc/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,3 @@
+<p>This project is the Traditional Chinese version of "霞鶩文楷". Initially, it mainly utilized AFDKO in conjunction with the legacy glyph shapes provided by the Zonz community to convert the characters contained in Klee One into old character forms, and supplemented the old version of "霞鶩文楷" with modifications. Some components and characters were further manually modified. Subsequently, reference was made to the "Legacy Glyph Standardization Document" from Yidianzifang to modify most components, making it more suitable for Traditional Chinese users and enthusiasts of legacy glyph forms.</p>
```

Comment:
@aaronbell, copy/paste the translation of the repo description here. Feel free to correct it directly here.

Reply:
Seemed fine to me! I just removed one word for better flow. Thanks

Reply:
> "Legacy Glyph Standardization Document" from Yidianzifang

The official document name is > "Inherited Glyph Standardization Documents" from I. Font Project <.
Quoted from https://github.com/ichitenfont/inheritedglyphs/blob/master/Readme_eng.md.